girder/girder_large_image/web_client/vue/components/CompositeLayers.vue

```diff
@@ -216,15 +321,116 @@ export default {
     top: 0px;
     background-color: white;
     z-index: 2;
+    border-bottom: 3px solid;
+}
+.small-text {
+    font-size: 10px;
+}
+.tall-row {
+    height: 75px;
+}
+.enable-col {
+    width: 10%;
+}
+.name-col {
+    width: 40%;
+}
+.color-col {
+    width: 25%;
+}
+.auto-range-col {
+    min-width: 100px;
+    display: flex;
+    column-gap: 10px;
+    align-content: space-around;
+    padding: 0;
+}
+.auto-range-label {
+    display: flex;
+    flex-direction: column;
+}
+.switch {
+  position: relative;
+  display: inline-block;
+  width: 45px;
+  height: 20px;
+  margin-top: 5px;
+}
+.slider {
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: #ccc;
+  -webkit-transition: .4s;
+  transition: .4s;
+  border-radius: 34px;
+}
+.slider.checked {
+  background-color: #2196F3;
+}
+.slider:focus{
+  box-shadow: 0 0 1px #2196F3;
+}
+.slider:before {
+  position: absolute;
+  content: "";
+  height: 15px;
+  width: 15px;
+  left: 4px;
+  bottom: 2px;
+  background-color: white;
+  -webkit-transition: .4s;
+  transition: .4s;
+  border-radius: 50%;
+}
+.slider.checked:before {
+  -webkit-transform: translateX(22px);
+  -ms-transform: translateX(22px);
+  transform: translateX(22px);
 }
 .table-container {
-    overflow-x: auto;
-    overflow-y: hidden;
+    overflow: auto;
+    max-height: 300px;
```

Comment:
This makes it surprising to edit the colors, as the color dialog gets hidden near the bottom.  Also, if you expand a row such that it causes the scroll bar to appear, then something about the histogram gets cropped and I can't grab the right control handle:
![image](https://github.com/girder/large_image/assets/8781639/d28cdb77-54b4-407a-b519-17998af1045e)
The red band was visible before expanding all, and here the right handle isn't grabbable  any more.

Reply:
For the color pickers, I made them relative to the table container in [0799c52](https://github.com/girder/large_image/pull/1175/commits/0799c52cf27bf2a1595217fd5be5107b5b906720). For the scrollbar/canvas resize conflict, my solution was to make the scrollbar always visible: [35601fc](https://github.com/girder/large_image/pull/1175/commits/35601fc8de52546db289484ca68a8588c229ff63)